developer/building/qubes-iso-building.md

```diff
@@ -19,6 +19,11 @@ Fedora 32 has been successfully used to build Qubes R4.1 with the below steps.
 Other rpm-based operating systems may also work.
 Travis-CI uses Ubuntu 18.04 to perform test builds, except it can not test the `./setup` script.
 
+**Notes:** On modern Fedora system (like Fedora 37) SeLinux is enforced by
+default and is blocking the build system. You would get error like 
+"can't create transaction lock on /.../rpm/.rpm.lock (Permission denied)". 
```

Comment:
Should a recommended workaround be included?

Reply:
The workaround is to switch selinux to permissive mode: `sudo setenforce 0`. And yes, I think it's a good idea to include it here.

Otherwise looks good.

Reply:
Ok :) added the workaround